yoyodyne/models/__init__.py

```diff
@@ -74,6 +76,7 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
             "pointer_generator_transformer",
             "transducer",
             "transformer",
+            "hmm_lstm",
```

Comment:
Whatever name you go with, put this list in lexicographic order like it was beforehand.